README.md

```diff
@@ -5,48 +5,31 @@
 
 ## Description
 
-Basic datatypes for writing code that handles poker (poker simulation, a poker website, solver etc.). Most datatypes and their functions are well-tested; untested functions are generally documented. The goal is to eventually reach 100% code coverage and for this package to be kept to a high industrial quality.
+Basic datatypes for writing code that handles poker (poker simulation, a poker website, solver etc.). This package is intended to be industry-ready (WIP).
```

Comment:
Just a suggested rewrite (complete sentences and no apologies(!) etc).

This library provides basic datatypes and infrastructure to handle poker computation. Once developed, the package intends to be industry-ready and provide capabilities that support poker websites, simulation, research and poker trainers and assistants.